tests/test_bucketlist.py

```diff
@@ -0,0 +1,100 @@
+"""test_bucketlist.py."""
+import json
+from tests.test_setup import BaseTestCase
+
+
+class BucketListTestCase(BaseTestCase):
+    """This class contains tests for the bucketlist."""
+
+    URL = "/api/v1/bucketlists/"
+
+    def test_createBucketList(self):
```

Comment:
This test function could be more descriptively named eg. test_create_new_bucketlist